gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.14-bin.zip
```

Comment:
I changed the gradle wrapper version to a more recent one because I was having issues getting it to work with the jdk version on my pc. I kept the change in because I think it makes sense to use a more recent version but I can remove the change if you like